### PR TITLE
VIH-7833 VHOs unable to add new panel members to hearing

### PR DESCRIPTION
**Pull request by mulvaneyjHMCTS**
### JIRA link (if applicable) ###
VIH-7833

### Change description ##
Removed Panel Member and Winger from the judiciary roles constant
Removed Panel Member and Winger from hearings controller check when booking a hearing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts [hunk 1]
```diff
@@ -1376,11 +1356,11 @@ describe('AddParticipantComponent edit mode no participants added', () => {
         expect(component.participantDetails).not.toBeNull();
         expect(component.participantDetails.username).toBeNull();
     });
-    it('should return JOH not found when role is Panel Member/Winger', () => {
-        role.setValue('Panel Member');
-        component.notFoundParticipant();
-        expect(component.errorJohAccountNotFound).toBeTruthy();
-    });
+    // it('should return JOH not found when role is Panel Member/Winger', () => {
```
Comment (robertbennett1998) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts:1359]: Does this need removing or adding back in?

**Review by robertbennett1998 — CHANGES_REQUESTED** [review]
LGTM. Just either fully remove or add back in that commented out code. Otherwise, leave a comment as to why it is commented out.

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts (hunk 1 above)
Comment (mulvaneyjHMCTS) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts]: Forgot to remove these, good spot.

**Review by robertbennett1998 — DISMISSED** [review]
LGTM

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts [hunk 2]
```diff
@@ -799,20 +793,6 @@ describe('AddParticipantComponent', () => {
...
             });
-
-            it('should have errorJudiciaryAccount set to true if search service returns null and role is Panel Member', () => {
-                searchServiceSpy.searchJudiciaryEntries.and.returnValue(of(null));
-                role.setValue('Panel Member'); // TODO fix magic string
-                component.validateJudiciaryEmailAndRole();
-                expect(component.errorJudiciaryAccount).toBeTruthy();
-            });
-
-            it('should have errorJudiciaryAccount set to true if search service returns null and role is Winger', () => {
```
Comment (FarronF) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts:810]: For whenever we need to readd this functionality these tests can probably stay gone as it looks like the previous tests do the same thing with test cases.

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts [hunk 3]
```diff
@@ -756,16 +756,10 @@ describe('AddParticipantComponent', () => {
             const testCases = [
                 { searchJudiciaryEntriesValue: null, role: '', expectError: false },
                 { searchJudiciaryEntriesValue: null, role: 'Other', expectError: false },
-                { searchJudiciaryEntriesValue: null, role: 'Panel Member', expectError: true },
-                { searchJudiciaryEntriesValue: null, role: 'Winger', expectError: true },
```
Comment (FarronF) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts:760]: Actually could you re-add these but just change the expectations. We probably should look at another way to list specific test cases we want to test

**Review by FarronF — CHANGES_REQUESTED** [review]

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts (hunk 3 above)
Comment (mulvaneyjHMCTS) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts:760]: Done

**Review by FarronF — APPROVED** [review]

**Pull request merged**